examples/pruning/tfkeras_integration.py

```diff
@@ -0,0 +1,149 @@
+"""
+Optuna example that demonstrates a pruner for tf.keras.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition
+using tf.keras and MNIST, where the architecture of the neural network
+and the parameters of optimizer are optimized.
+Throughout the training of neural networks,
+a pruner observes intermediate results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python tfkeras_integration.py
+
+"""
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+import optuna
+from optuna.integration import TFKerasPruningCallback
+
+
+BATCHSIZE = 128
+CLASSES = 10
+EPOCHS = 20
+N_TRAIN_EXAMPLES = 60000
+STEPS_PER_EPOCH = int(N_TRAIN_EXAMPLES / BATCHSIZE / 10)
+VALIDATION_STEPS = 30
+
+
+def train_dataset():
+    # type: () -> tf.data.Dataset
```

Comment:
Please remove type hints in this file. 
Optuna's examples do not have type hints for novice users to read the code easily.

Reply:
I fixed it